eng/common/pipelines/templates/steps/prepare-pipelines.yml

```diff
@@ -74,6 +77,25 @@ steps:
       condition: ne('${{parameters.TestsConventionOptions}}','')
       env:
         PATVAR: $(azuresdk-azure-sdk-devops-pipeline-generation-pat)
+    - script: >
```

Comment:
Not sure we care at this point but if we want this for the private repo as well we would need to add it to the section below also.

Reply:
I don't think we do, but do you think I should add it for consistency?

Reply:
It probably is worth having for consistency, also we tend to use the prepare-pipelines stuff for one-off pipelines we setup in private repos (or at least I'd like us to).